spec/rails_helper.rb

```diff
@@ -59,3 +59,10 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
```

Comment:
This is my fault somewhat, but [Simon's PR](https://github.com/alphagov/local-links-manager/pull/4/files) adds shoulda-matchers already so we'll have a conflict to resolve when his is merged and we re-work this one on top of it.  If we're lucky git will resolve it, but we should check that we haven't duplicated this config, particularly as Simon's version lives in `spec/support/shoulda.rb` instead of `spec/rails_helper.rb` (it's better to keep `spec/spec_helper.rb` and `spec/rails_helper.rb` as pristine as possible to make it easier to incorporate changes to these files when we upgrade rspec and rspec-rails).

There's nothing we need to do now just something to be aware of when we rework this once Simon's is merged.